en/docs/learn/api-controller/advanced-topics/configuring-git-integration.md

```diff
@@ -0,0 +1,96 @@
+#  Configuring Git Integration
+
+API Controller natively supports identifying API or any other types of projects in a Git repository and seamlessly deploying the changes of the projects to a given API Manager environment. This topic is regarding some of the configurations that you can use to optimize the Git integration functionality.
```

Comment:
Shall we modify as "Identifying APIs"?

Reply:
What I meant here was API as project type.. which one would be better here?
- API or any other types of projects
- API or any other type of projects
- APIs or any other types of projects

Reply:
will go with above "Identifying APIs". That looks better.